x-pack/plugins/index_lifecycle_management/public/store/actions/index.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+
+export * from './index_template';
+export * from './nodes';
+export * from './policies';
+export * from './lifecycle';
+export * from './general';
```

Comment:
Can we avoid exporting `*` and instead use named re-exports? I find this makes it easier to look in a single place and see which actions are available. Maybe it's just me though. 😄

Reply:
I think that just leads to maintenance down the road.  As you add things to the files, you have to do everything twice.